pep-0008.txt

```diff
@@ -850,6 +850,30 @@ names.
 In some fonts, these characters are indistinguishable from the
 numerals one and zero.  When tempted to use 'l', use 'L' instead.
 
+ASCII Compatibility
+~~~~~~~~~~~~~~~~~~~
+
+While support for Unicode identifiers has been added to allow all
+Python developers to use variable names in their native languages,
+the 7-bit ASCII text encoding remains the lowest common denominator
+for text entry when developing Python programs.
```

Comment:
This entire paragraph is explanation that is widely known and doesn't need to be repeated in PEP 8.